starter-code/server.js

```diff
@@ -24,7 +24,8 @@ app.get('/new', function(request, response) {
 app.get('/articles', function(request, response) {
   // REVIEW: This query will join the data together from our tables and send it back to the client.
   // TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
-  client.query(``)
+
+  client.query(`SELECT * FROM article JOIN on article.author_id=authors.author_id`)
```

Comment:
table name is `articles` not `article`
you are missing the table name you want to join article to
```
SELECT *
FROM articles
JOIN authors
ON articles.author_id=authors.author_id
```